fed_test_utils/fed.go

```diff
@@ -153,51 +154,89 @@ func NewFedTest(t *testing.T, originConfig string) (ft *FedTest) {
 	directorStartTime := time.Now().Add(-6 * time.Minute)
 	director.SetStartupTime(directorStartTime)
 
-	config.InitConfig()
 	err = config.InitServer(ctx, modules)
 	require.NoError(t, err)
 
 	// Read in any config we may have set
+	var originConf interface{}
 	if originConfig != "" {
 		viper.SetConfigType("yaml")
 		err = viper.MergeConfig(strings.NewReader(originConfig))
 		require.NoError(t, err, "error reading config")
+
+		err := yaml.Unmarshal([]byte(originConfig), &originConf)
+		require.NoError(t, err, "error unmarshalling into interface")
+
+		confMap := originConf.(map[string]interface{})
+
+		originRaw, exists := confMap["Origin"]
+
+		if exists {
+			originMap := originRaw.(map[string]interface{})
+
+			// Override the test directory from the config file with our temp directory
+			if exportsRaw, ok := originMap["Exports"]; ok {
```

Comment:
In the top-level if, you use `exists` instead of `ok` to determine whether the entry is in the map -- I think `ok` is more canonically "go", but I honestly think "exists" is easier to read. Can you pick one or the other and use in both cases?